pkgs/development/python-modules/flet-core/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, python3
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "flet-core";
+  version = "0.6.2";
+  format = "pyproject";
+
+  src = fetchPypi {
+    pname = "flet_core";
+    inherit version;
+    hash = "sha256-WMkm+47xhuYz1HsiPfF7YbOCg7Xlbj9oHI9nVtwAb/w=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    typing-extensions
+    repath
+  ];
```

Comment:
We can't use `python3.pkgs` in the python package set. Instead, we need to pass all python packages via function arguments, otherwise you are pinning this to just one python version.

I'm fixing this up in #244135.

```
$ nom-build -A python311Packages.flet
error:
       … while evaluating the attribute 'drvPath'

         at /home/hexa/git/nixos/master/lib/customisation.nix:222:7:

          221|     in commonAttrs // {
          222|       drvPath = assert condition; drv.drvPath;
             |       ^
          223|       outPath = assert condition; drv.outPath;

       … while evaluating the attribute 'drvPath'

         at /home/hexa/git/nixos/master/lib/customisation.nix:222:7:

          221|     in commonAttrs // {
          222|       drvPath = assert condition; drv.drvPath;
             |       ^
          223|       outPath = assert condition; drv.outPath;

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: Python version mismatch in 'python3.11-flet-0.7.4':

       The Python derivation 'python3.11-flet-0.7.4' depends on a Python derivation
       named 'python3.10-flet-core-0.7.4', but the two derivations use different versions
       of Python:

                'python3.11-flet-0.7.4' uses /nix/store/ng1c2jqy48p1x33j1qyg0n5anhfv31g0-python3-3.11.4
           'python3.10-flet-core-0.7.4' uses /nix/store/jhflvwr40xbb0xr6jx4311icp9cym1fp-python3-3.10.12

       Possible solutions:

         * If 'python3.10-flet-core-0.7.4' is a Python library, change the reference to 'python3.10-flet-core-0.7.4'
           in the propagatedBuildInputs of 'python3.11-flet-0.7.4' to use a 'python3.10-flet-core-0.7.4' built from the same
           version of Python

         * If 'python3.10-flet-core-0.7.4' is used as a tool during the build, move the reference to
           'python3.10-flet-core-0.7.4' in 'python3.11-flet-0.7.4' from propagatedBuildInputs to nativeBuildInputs

         * If 'python3.10-flet-core-0.7.4' provides executables that are called at run time, pass its
           bin path to makeWrapperArgs:

               makeWrapperArgs = [ "--prefix PATH : ${lib.makeBinPath [ flet-core ] }" ];

        at /home/hexa/git/nixos/master/pkgs/development/python-modules/flet/default.nix:8:3

Reply:
Fixed in ae735c533cd091ff54df7425c6695edba08fede1 ddc1932de9bd88c5ea2fd9a54d3ac68270057590 c7437c5836ae7b0c9436d3b468bbba0197c524ee and two more in e66ea733b0374cff001e3132cf366214e60a75f0 and 92d6b02a3399172d9cd604d04847bc8fdb52a7ad